internal/commands/app/create_inputs.go

```diff
@@ -206,6 +208,7 @@ func (i *createInputs) resolveClusters(ui terminal.UI, client atlas.Client, grou
 					ReadPreference:      "primary",
 					WireProtocolEnabled: false,
 				},
+				Version: 1,
```

Comment:
I needed to make this 1 in order to pass validation for the import handler.

I'm guessing that manually setting 1 here is not optimal. I'm having trouble finding context for what this version is, and how its being used.

Is there a better way of getting this?

Reply:
Setting to 1 is fine, afaik this only relates to using promises.
https://github.com/10gen/baas/blob/master/service/models/version.go

https://github.com/10gen/baas/blob/master/function/execution/vm/stream.go#L33